src/classes/FORM_PermissionValidator.cls

```diff
@@ -0,0 +1,146 @@
+public with sharing class FORM_PermissionValidator {
```

Comment:
Add javadoc style header comment for the class.

Reply:
fixed